File: WebApp/Scripts/app/model/BD0002M.js
Ext.define('WEBAPP.model.BD0002M', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'PR_NO', type: 'string' }, //申購單號
        { name: 'PR_TIME', type: 'string' }, //申購時間
        { name: 'MAT_CLASS', type: 'string' }, //物料類別
        { name: 'M_STOREID', type: 'string' }, //庫備識別碼
        { name: 'M_STOREID_NAME', type: 'string' }, //庫備識別碼名稱
        { name: 'M_STOREID_CODE', type: 'string' }, //庫備識別碼+名稱
        { name: 'PR_STATUS', type: 'string' }, //申購單狀態代碼
        { name: 'PR_STATUS_NAME', type: 'string' }, //申購單狀態名稱
        { name: 'PR_STATUS_CODE', type: 'string' }, //申購單狀態代碼+名稱
        { name: 'XACTION', type: 'string' }, //申購類別代碼
        { name: 'XACTION_NAME', type: 'string' }, //申購類別名稱
        { name: 'XACTION_CODE', type: 'string' } //申購類別代碼+名稱
    ]
});

File: WebApp/Models/BD/BD0002M.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class BD0002M : JCLib.Mvc.BaseModel
    {
        public string PR_NO { get; set; } //申購單號
        public string PR_TIME { get; set; } //申購時間
        public string MAT_CLASS { get; set; } //物料類別
        public string M_STOREID { get; set; } //庫備識別碼
        public string M_STOREID_NAME { get; set; } //庫備識別碼名稱
        public string M_STOREID_CODE { get; set; } //庫備識別碼+名稱
        public string PR_STATUS { get; set; } //申購單狀態代碼
        public string PR_STATUS_NAME { get; set; } //申購單狀態名稱
        public string PR_STATUS_CODE { get; set; } //申購單狀態代碼+名稱
        public string XACTION { get; set; } //申購類別代碼
        public string XACTION_NAME { get; set; } //申購類別名稱
        public string XACTION_CODE { get; set; } //申購類別代碼+名稱

        public string IS_CR { get; set; }
    }
}
